2023-12-08 19:25:49,876 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 19:25:50,962 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.model_evaluation' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/pipeline.py", line 7, in <module>
    from .nodes import train_model,evaluate_model
ImportError: cannot import name 'train_model' from 'sales_project.pipelines.model_evaluation.nodes' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/nodes.py)

  warnings.warn(

2023-12-08 19:32:57,986 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 19:32:58,680 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.model_evaluation' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/pipeline.py", line 7, in <module>
    from .nodes import train_model,evaluate_model
ImportError: cannot import name 'train_model' from 'sales_project.pipelines.model_evaluation.nodes' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/nodes.py)

  warnings.warn(

2023-12-08 19:39:04,829 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 19:39:05,610 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.model_evaluation' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/pipeline.py", line 7, in <module>
    from .nodes import train_model,evaluate_model
ImportError: cannot import name 'train_model' from 'sales_project.pipelines.model_evaluation.nodes' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/nodes.py)

  warnings.warn(

2023-12-08 19:53:33,403 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 19:53:34,135 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.model_evaluation' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/pipeline.py", line 7, in <module>
    from .nodes import train_model,evaluate_model
ImportError: cannot import name 'train_model' from 'sales_project.pipelines.model_evaluation.nodes' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/model_evaluation/nodes.py)

  warnings.warn(

2023-12-08 19:53:34,447 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 19:53:34,486 - kedro.pipeline.node - INFO - Running node: data_ingestion([sale_data]) -> [preprocessed_data]
2023-12-08 19:53:34,489 - kedro.pipeline.node - ERROR - Node 'data_ingestion([sale_data]) -> [preprocessed_data]' failed with error: 
argument of type 'method' is not iterable
2023-12-08 19:53:34,499 - kedro.runner.sequential_runner - WARNING - No nodes ran. Repeat the previous command to attempt a new run.
2023-12-08 20:06:31,135 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:06:32,212 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:06:32,252 - kedro.pipeline.node - INFO - Running node: data_ingestion([sale_data]) -> [preprocessed_data]
2023-12-08 20:06:32,255 - kedro.pipeline.node - ERROR - Node 'data_ingestion([sale_data]) -> [preprocessed_data]' failed with error: 
argument of type 'method' is not iterable
2023-12-08 20:06:32,267 - kedro.runner.sequential_runner - WARNING - No nodes ran. Repeat the previous command to attempt a new run.
2023-12-08 20:13:25,760 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:14:02,481 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:14:28,003 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:15:18,353 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:16:11,413 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:22:23,455 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:22:24,402 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:22:24,449 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:22:24,489 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:22:24,536 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:22:24,551 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:22:24,558 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:22:24,591 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:22:24,635 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:22:24,658 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:22:24,671 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:22:24,677 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:22:24,681 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:22:24,682 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:22:24,697 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:22:24,700 - kedro.io.data_catalog - INFO - Loading data from 'params:model_params' (MemoryDataset)...
2023-12-08 20:22:24,714 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train,params:model_params]) -> [model]
2023-12-08 20:22:24,717 - kedro.pipeline.node - ERROR - Node 'train_model([X_train,y_train,params:model_params]) -> [model]' failed with error: 
RandomForestRegressor.__init__() got an unexpected keyword argument 'penalty'
2023-12-08 20:22:24,730 - kedro.runner.sequential_runner - WARNING - There are 2 nodes that have not run.
You can resume the pipeline run from the nearest nodes with persisted inputs by adding the following argument to your previous command:
  --from-nodes "split_data([processed_data]) -> [X_train,X_test,y_train,y_test],evaluate_model([model,X_test,y_test]) -> None"
2023-12-08 20:25:17,739 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:25:18,696 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:25:18,746 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:25:18,773 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:25:18,825 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:25:18,840 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:25:18,842 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:25:18,867 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:25:18,912 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:25:18,934 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:25:18,950 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:25:18,953 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:25:18,956 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:25:18,957 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:25:18,975 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:25:18,977 - kedro.io.data_catalog - INFO - Loading data from 'params:model_params' (MemoryDataset)...
2023-12-08 20:25:18,989 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train,params:model_params]) -> [model]
2023-12-08 20:25:18,990 - kedro.pipeline.node - ERROR - Node 'train_model([X_train,y_train,params:model_params]) -> [model]' failed with error: 
'penality'
2023-12-08 20:25:19,003 - kedro.runner.sequential_runner - WARNING - There are 2 nodes that have not run.
You can resume the pipeline run from the nearest nodes with persisted inputs by adding the following argument to your previous command:
  --from-nodes "split_data([processed_data]) -> [X_train,X_test,y_train,y_test],evaluate_model([model,X_test,y_test]) -> None"
2023-12-08 20:26:13,869 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:26:14,778 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:26:14,823 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:26:14,857 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:26:14,907 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:26:14,924 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:26:14,927 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:26:14,952 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:26:14,997 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:26:15,018 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:26:15,030 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:26:15,034 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:26:15,037 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:26:15,039 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:26:15,051 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:26:15,054 - kedro.io.data_catalog - INFO - Loading data from 'params:model_params' (MemoryDataset)...
2023-12-08 20:26:15,067 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train,params:model_params]) -> [model]
2023-12-08 20:26:15,069 - kedro.pipeline.node - ERROR - Node 'train_model([X_train,y_train,params:model_params]) -> [model]' failed with error: 
RandomForestRegressor.__init__() got an unexpected keyword argument 'class_weight'
2023-12-08 20:26:15,084 - kedro.runner.sequential_runner - WARNING - There are 2 nodes that have not run.
You can resume the pipeline run from the nearest nodes with persisted inputs by adding the following argument to your previous command:
  --from-nodes "evaluate_model([model,X_test,y_test]) -> None,split_data([processed_data]) -> [X_train,X_test,y_train,y_test]"
2023-12-08 20:27:06,486 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:27:34,268 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:27:35,216 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:27:35,253 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:27:35,285 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:27:35,331 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:27:35,346 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:27:35,348 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:27:35,374 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:27:35,420 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:27:35,443 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:27:35,457 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:27:35,463 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:27:35,466 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:27:35,468 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:27:35,482 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:27:35,494 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-08 20:27:36,295 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-08 20:27:36,311 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-08 20:27:36,312 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-08 20:27:36,320 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-08 20:27:36,335 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-08 20:27:36,350 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> None
2023-12-08 20:27:36,364 - kedro.pipeline.node - ERROR - Node 'evaluate_model([model,X_test,y_test]) -> None' failed with error: 
local variable 'r2_score' referenced before assignment
2023-12-08 20:27:36,379 - kedro.runner.sequential_runner - WARNING - There are 1 nodes that have not run.
You can resume the pipeline run from the nearest nodes with persisted inputs by adding the following argument to your previous command:
  --from-nodes "evaluate_model([model,X_test,y_test]) -> None"
2023-12-08 20:28:11,716 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:28:12,724 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:28:12,766 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:28:12,791 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:28:12,832 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:28:12,845 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:28:12,849 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:28:12,875 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:28:12,924 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:28:12,944 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:28:12,957 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:28:12,960 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:28:12,963 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:28:12,965 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:28:12,977 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:28:12,992 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-08 20:28:13,774 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-08 20:28:13,792 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-08 20:28:13,794 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-08 20:28:13,802 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-08 20:28:13,816 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-08 20:28:13,829 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> None
2023-12-08 20:28:13,852 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-08 20:28:13,853 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-08 20:36:03,314 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:36:04,257 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:36:04,299 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:36:04,329 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:36:04,381 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:36:04,396 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:36:04,397 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:36:04,420 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:36:04,466 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:36:04,488 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:36:04,505 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:36:04,511 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:36:04,514 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:36:04,516 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:36:04,527 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:36:04,540 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-08 20:36:05,219 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-08 20:36:05,238 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-08 20:36:05,240 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-08 20:36:05,249 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-08 20:36:05,260 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-08 20:36:05,273 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> None
2023-12-08 20:36:05,296 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-08 20:36:05,297 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-08 20:36:54,507 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:36:55,478 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:36:55,518 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:36:55,549 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:36:55,591 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:36:55,608 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:36:55,610 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:36:55,636 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:36:55,679 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:36:55,698 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:36:55,708 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:36:55,712 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:36:55,716 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:36:55,717 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:36:55,728 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:36:55,739 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-08 20:36:56,520 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-08 20:36:56,543 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-08 20:36:56,546 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-08 20:36:56,554 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-08 20:36:56,569 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-08 20:36:56,581 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> None
2023-12-08 20:36:56,602 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-08 20:36:56,603 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-08 20:38:03,123 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-08 20:38:04,091 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-08 20:38:04,128 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-08 20:38:04,155 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-08 20:38:04,201 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-08 20:38:04,235 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-08 20:38:04,236 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-08 20:38:04,261 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-08 20:38:04,304 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-08 20:38:04,324 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-08 20:38:04,335 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-08 20:38:04,339 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-08 20:38:04,342 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-08 20:38:04,344 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-08 20:38:04,354 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-08 20:38:04,365 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-08 20:38:04,467 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-08 20:38:04,473 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-08 20:38:04,475 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-08 20:38:04,477 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-08 20:38:04,487 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-08 20:38:04,498 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> None
2023-12-08 20:38:04,519 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-08 20:38:04,521 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 12:23:15,695 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-09 13:17:18,263 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 13:17:19,531 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-09 13:17:19,720 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 13:17:19,767 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 13:17:19,814 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 13:17:19,837 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 13:17:19,839 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 13:17:19,866 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 13:17:19,917 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 13:17:19,943 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 13:17:19,961 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 13:17:19,966 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 13:17:19,972 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 13:17:19,974 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 13:17:19,987 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 13:17:20,001 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 13:17:20,093 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 13:17:20,101 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 13:17:20,103 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 13:17:20,108 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 13:17:20,121 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 13:17:20,134 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 13:17:20,143 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 13:17:20,157 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 13:17:20,163 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 13:17:20,165 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 13:17:56,998 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-09 13:25:41,020 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 13:30:31,367 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 13:35:04,291 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 13:42:29,563 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 13:44:45,567 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 13:48:53,138 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:19:23,917 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:19:53,912 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:22:05,020 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:22:05,991 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-09 14:23:07,243 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:23:08,239 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-09 14:34:59,176 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:35:00,177 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py:366: UserWarning: An error occurred while importing the 'sales_project.pipelines.data_ingestion' module. Nothing defined therein will be returned by 'find_pipelines'.

Traceback (most recent call last):
  File "/home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/project/__init__.py", line 364, in find_pipelines
    pipeline_module = importlib.import_module(pipeline_module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/__init__.py", line 6, in <module>
    from .pipeline import create_pipeline
ImportError: cannot import name 'create_pipeline' from 'sales_project.pipelines.data_ingestion.pipeline' (/home/unibash/G3/B/SALE_CNV_OPT/sales-project/src/sales_project/pipelines/data_ingestion/pipeline.py)

  warnings.warn(

2023-12-09 14:37:43,454 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:44:27,086 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:45:32,121 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:46:00,231 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:46:01,515 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 14:46:01,555 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 14:46:01,604 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 14:46:01,625 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 14:46:01,628 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 14:46:01,656 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 14:46:01,707 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 14:46:01,729 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 14:46:01,742 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 14:46:01,746 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 14:46:01,750 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 14:46:01,751 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 14:46:01,767 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 14:46:01,780 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 14:46:01,874 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 14:46:01,881 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 14:46:01,882 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 14:46:01,885 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 14:46:01,896 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 14:46:01,909 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 14:46:01,916 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 14:46:01,940 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 14:46:01,946 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 14:46:01,947 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 14:52:00,353 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 14:52:01,704 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 14:52:01,735 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 14:52:01,782 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 14:52:01,810 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 14:52:01,812 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 14:52:01,851 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 14:52:01,908 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 14:52:01,929 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 14:52:01,942 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 14:52:01,946 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 14:52:01,949 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 14:52:01,951 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 14:52:01,963 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 14:52:01,976 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 14:52:02,072 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 14:52:02,078 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 14:52:02,080 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 14:52:02,083 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 14:52:02,100 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 14:52:02,113 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 14:52:02,130 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 14:52:02,147 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 14:52:02,151 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 14:52:02,152 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 15:27:57,709 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 15:27:59,079 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 15:27:59,113 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 15:27:59,161 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 15:27:59,182 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 15:27:59,184 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 15:27:59,214 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 15:27:59,265 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 15:27:59,286 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 15:27:59,298 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 15:27:59,303 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 15:27:59,307 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 15:27:59,308 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 15:27:59,320 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 15:27:59,334 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 15:27:59,510 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 15:27:59,518 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 15:27:59,520 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 15:27:59,523 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 15:27:59,535 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 15:27:59,548 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 15:27:59,556 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.756 on test data.
2023-12-09 15:27:59,571 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 15:27:59,576 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 15:27:59,578 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 15:28:15,570 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 15:28:16,948 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 15:28:16,979 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 15:28:17,021 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 15:28:17,042 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 15:28:17,044 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 15:28:17,073 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 15:28:17,121 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 15:28:17,147 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 15:28:17,164 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 15:28:17,170 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 15:28:17,174 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 15:28:17,176 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 15:28:17,188 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 15:28:17,203 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 15:28:17,375 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 15:28:17,382 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 15:28:17,384 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 15:28:17,387 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 15:28:17,402 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 15:28:17,416 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 15:28:17,422 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.756 on test data.
2023-12-09 15:28:17,435 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 15:28:17,438 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 15:28:17,441 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 15:28:28,041 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 15:28:29,298 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 15:28:29,328 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 15:28:29,377 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 15:28:29,396 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 15:28:29,398 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 15:28:29,424 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 15:28:29,483 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 15:28:29,505 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 15:28:29,518 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 15:28:29,521 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 15:28:29,524 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 15:28:29,526 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 15:28:29,539 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 15:28:29,553 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 15:28:29,643 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 15:28:29,648 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 15:28:29,649 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 15:28:29,652 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 15:28:29,662 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 15:28:29,675 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 15:28:29,681 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 15:28:29,699 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 15:28:29,706 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 15:28:29,708 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 15:58:26,579 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 15:58:26,585 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 15:58:27,854 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 15:58:27,894 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 15:58:27,948 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 15:58:27,974 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 15:58:27,977 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 15:58:28,015 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 15:58:28,067 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 15:58:28,093 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 15:58:28,108 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 15:58:28,114 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 15:58:28,117 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 15:58:28,118 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 15:58:28,133 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 15:58:28,148 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 15:58:28,235 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 15:58:28,241 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 15:58:28,242 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 15:58:28,245 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 15:58:28,257 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 15:58:28,270 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 15:58:28,276 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 15:58:28,291 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 15:58:28,295 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 15:58:28,297 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 15:58:50,870 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 17:23:05,033 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 17:23:05,037 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 17:23:06,064 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 17:23:06,066 - kedro.runner.sequential_runner - WARNING - No nodes ran. Repeat the previous command to attempt a new run.
2023-12-09 17:24:30,199 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 17:24:30,202 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 17:24:30,865 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 17:24:30,884 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 17:24:30,907 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 17:24:30,916 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 17:24:30,917 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 17:24:30,932 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 17:24:30,957 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 17:24:30,970 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 17:24:30,979 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 17:24:30,981 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 17:24:30,982 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 17:24:30,983 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 17:24:30,989 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 17:24:30,995 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 17:24:31,043 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 17:24:31,047 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 17:24:31,048 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 17:24:31,050 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 17:24:31,056 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 17:24:31,065 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 17:24:31,069 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 17:24:31,076 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 17:24:31,080 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 17:24:31,081 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 18:30:30,422 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 18:30:30,429 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 18:30:32,451 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 18:30:32,454 - kedro.runner.sequential_runner - WARNING - No nodes ran. Repeat the previous command to attempt a new run.
2023-12-09 18:31:01,529 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 18:31:01,537 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 18:31:02,811 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 18:31:02,856 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 18:31:02,908 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 18:31:02,928 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 18:31:02,929 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 18:31:02,955 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 18:31:03,003 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 18:31:03,032 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 18:31:03,045 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 18:31:03,050 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 18:31:03,053 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 18:31:03,055 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 18:31:03,067 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 18:31:03,080 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 18:31:03,176 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 18:31:03,180 - kedro.runner.sequential_runner - WARNING - There are 2 nodes that have not run.
You can resume the pipeline run from the nearest nodes with persisted inputs by adding the following argument to your previous command:
  --from-nodes "evaluate_model([model,X_test,y_test]) -> [metrics],train_model([X_train,y_train]) -> [model]"
2023-12-09 18:32:24,326 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 18:32:24,331 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 18:32:25,580 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 18:32:25,613 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 18:32:25,660 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 18:32:25,676 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 18:32:25,678 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 18:32:25,706 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 18:32:25,750 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 18:32:25,771 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 18:32:25,784 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 18:32:25,788 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 18:32:25,792 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 18:32:25,794 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 18:32:25,805 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 18:32:25,818 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 18:32:25,907 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 18:32:25,918 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 18:32:25,920 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 18:32:25,922 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 18:32:25,938 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 18:32:25,952 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 18:32:25,958 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 18:32:25,975 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 18:32:25,981 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 18:32:25,983 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 18:32:45,035 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 18:32:45,042 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 18:32:46,259 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 18:32:46,286 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 18:32:46,332 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 18:32:46,349 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 18:32:46,351 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 18:32:46,379 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 18:32:46,421 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 18:32:46,444 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 18:32:46,461 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 18:32:46,466 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 18:32:46,469 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 18:32:46,471 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 18:32:46,483 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 18:32:46,496 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 18:32:46,584 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 18:32:46,591 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 18:32:46,592 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 18:32:46,595 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 18:32:46,606 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 18:32:46,620 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 18:32:46,628 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 18:32:46,642 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 18:32:46,649 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 18:32:46,650 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 18:33:00,876 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 18:33:00,881 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 18:33:02,046 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 18:33:02,074 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 18:33:02,120 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 18:33:02,137 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 18:33:02,139 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 18:33:02,165 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 18:33:02,209 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 18:33:02,230 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 18:33:02,244 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 18:33:02,248 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 18:33:02,253 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 18:33:02,254 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 18:33:02,266 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 18:33:02,279 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 18:33:02,368 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 18:33:02,374 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 18:33:02,376 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 18:33:02,379 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 18:33:02,394 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 18:33:02,409 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 18:33:02,416 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 18:33:02,430 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 18:33:02,434 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 18:33:02,436 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 19:28:54,018 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 19:31:03,738 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 19:31:03,745 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 19:31:05,062 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 19:31:05,096 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 19:31:05,142 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 19:31:05,165 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 19:31:05,167 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 19:31:05,196 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 19:31:05,242 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 19:31:05,265 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 19:31:05,279 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 19:31:05,286 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 19:31:05,289 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 19:31:05,291 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 19:31:05,305 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 19:31:05,319 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 19:31:05,409 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 19:31:05,416 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 19:31:05,418 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 19:31:05,421 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 19:31:05,435 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 19:31:05,449 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 19:31:05,456 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 19:31:05,472 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 19:31:05,477 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 19:31:05,478 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 19:31:21,688 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 21:34:52,399 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 21:34:52,404 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 21:34:54,438 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 21:34:54,473 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 21:34:54,525 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 21:34:54,555 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 21:34:54,557 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 21:34:54,593 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 21:34:54,645 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 21:34:54,674 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 21:34:54,690 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 21:34:54,696 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 21:34:54,699 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 21:34:54,700 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 21:34:54,713 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 21:34:54,726 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 21:34:54,841 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 21:34:54,848 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 21:34:54,850 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 21:34:54,854 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 21:34:54,871 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 21:34:54,884 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 21:34:54,919 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 21:34:54,943 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 21:34:54,951 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 21:34:54,952 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
2023-12-09 21:35:51,333 - kedro.framework.session.session - INFO - Kedro project sales-project
2023-12-09 21:35:51,339 - py.warnings - WARNING - /home/unibash/G3/B/SALE_CNV_OPT/SCC/lib/python3.10/site-packages/kedro/framework/session/session.py:267: KedroDeprecationWarning: ConfigLoader will be deprecated in Kedro 0.19. Please use the OmegaConfigLoader instead. To consult the documentation for OmegaConfigLoader, see here: https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#omegaconfigloader
  warnings.warn(

2023-12-09 21:35:53,128 - kedro.io.data_catalog - INFO - Loading data from 'sale_data' (CSVDataSet)...
2023-12-09 21:35:53,156 - kedro.pipeline.node - INFO - Running node: preprocessing([sale_data]) -> [processed_data]
2023-12-09 21:35:53,209 - kedro.io.data_catalog - INFO - Saving data to 'processed_data' (PickleDataSet)...
2023-12-09 21:35:53,229 - kedro.runner.sequential_runner - INFO - Completed 1 out of 4 tasks
2023-12-09 21:35:53,231 - kedro.io.data_catalog - INFO - Loading data from 'processed_data' (PickleDataSet)...
2023-12-09 21:35:53,261 - kedro.pipeline.node - INFO - Running node: split_data([processed_data]) -> [X_train,X_test,y_train,y_test]
2023-12-09 21:35:53,316 - kedro.io.data_catalog - INFO - Saving data to 'X_train' (PickleDataSet)...
2023-12-09 21:35:53,338 - kedro.io.data_catalog - INFO - Saving data to 'X_test' (PickleDataSet)...
2023-12-09 21:35:53,353 - kedro.io.data_catalog - INFO - Saving data to 'y_train' (PickleDataSet)...
2023-12-09 21:35:53,356 - kedro.io.data_catalog - INFO - Saving data to 'y_test' (PickleDataSet)...
2023-12-09 21:35:53,359 - kedro.runner.sequential_runner - INFO - Completed 2 out of 4 tasks
2023-12-09 21:35:53,361 - kedro.io.data_catalog - INFO - Loading data from 'X_train' (PickleDataSet)...
2023-12-09 21:35:53,375 - kedro.io.data_catalog - INFO - Loading data from 'y_train' (PickleDataSet)...
2023-12-09 21:35:53,389 - kedro.pipeline.node - INFO - Running node: train_model([X_train,y_train]) -> [model]
2023-12-09 21:35:53,475 - kedro.io.data_catalog - INFO - Saving data to 'model' (PickleDataSet)...
2023-12-09 21:35:53,482 - kedro.runner.sequential_runner - INFO - Completed 3 out of 4 tasks
2023-12-09 21:35:53,485 - kedro.io.data_catalog - INFO - Loading data from 'model' (PickleDataSet)...
2023-12-09 21:35:53,489 - kedro.io.data_catalog - INFO - Loading data from 'X_test' (PickleDataSet)...
2023-12-09 21:35:53,501 - kedro.io.data_catalog - INFO - Loading data from 'y_test' (PickleDataSet)...
2023-12-09 21:35:53,512 - kedro.pipeline.node - INFO - Running node: evaluate_model([model,X_test,y_test]) -> [metrics]
2023-12-09 21:35:53,519 - sales_project.pipelines.model_evaluation.nodes - INFO - Model has a coefficient R^2 of 0.754 on test data.
2023-12-09 21:35:53,537 - kedro.io.data_catalog - INFO - Saving data to 'metrics' (MetricsDataSet)...
2023-12-09 21:35:53,544 - kedro.runner.sequential_runner - INFO - Completed 4 out of 4 tasks
2023-12-09 21:35:53,545 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
